client/landing/gutenboarding/components/locale-context/index.tsx

```diff
@@ -31,7 +31,7 @@ interface AppWindow extends Window {
 	installedChunks?: string[];
 	i18nLocaleStrings?: string;
 	__requireChunkCallback__?: {
-		add( callback: Function ): void;
+		add( callback: Function ): void; // eslint-disable-line @typescript-eslint/ban-types
```

Comment:
This is a style issue I'm grandfathering in. Sorry.

Reply:
Fair enough. I was trying to figure out what the actual type of `callback` was, but it looks complicated and not worth it.

TBH another option is just to delete line 34, because we never actually use the `add` method ...